packages/manager/cypress/e2e/core/linodes/smoke-delete-linode.spec.ts

```diff
@@ -50,10 +108,59 @@ describe('delete linode', () => {
             .click();
         });
 
-      // confirm delete
+      // Confirm delete
       cy.wait('@deleteLinode').its('response.statusCode').should('eq', 200);
-      cy.url().should('endWith', '/linodes');
-      cy.findByText(linode.label).should('not.exist');
+      confirmDeletion(linode.label);
+    });
+  });
+
+  it('deletes linode from setting tab inlinode details page', () => {
+    createLinode().then((linode) => {
```

Comment:
```suggestion
    const linodeCreatePayload = createLinodeRequestFactory.build();
    cy.defer(createLinode(linodeCreatePayload)).then((linode) => {
```
Now that `createLinode()` is being imported from the API v4 package, you'll need to wrap these existing calls to `createLinode()` in `cy.defer()` since they return Promises now -- otherwise the test immediately passes without waiting for the Promise to resolve.

(This also applies to lines `72` and `168`)